docs/ml_get_event_certainty.md

```diff
@@ -0,0 +1,31 @@
+# on ML start
+
+Gets the latest certainty value for an ML action.
+
+```sig
+ml.getCertainty(ml.event.Unknown)
+```
+
+The ML model runs several times a second and calculates a certainty value for each action. The estimated action is the action with the highest certainty. An action cannot be the estimated action when its certainty is below the recognition point. Some programs may need to use the certainty values directly, for example to display or log them. Most programs can use the estimated action instead of certainty values.
+
+## Parameters
+
+- **event**: one of the actions the machine learning model was trained on.
+
+## Returns
+
+- a percentage as a [number](/types/number) from 0 to 100, representing the ML model’s certainty that this is the action being performed. The certainty for `unknown` is always 0.
+
+## Example
+
+This example displays the ML model's certainty that the current action is `unknown` every second.
+
+```blocks
+loops.everyInterval(1000, function () {
+    basic.showNumber(ml.getCertainty(ml.event.Unknown))
+})
+```
+
+```package
+machine-learning=github:microbit-foundation/pxt-microbit-ml
```

Comment:
Not sure whether this should in some way be versioned? Might be awkward or something we can revisit after getting this documentation out.

Reply:
Target most recently tagged version for now [69336c5](https://github.com/microbit-foundation/pxt-microbit-ml/pull/29/commits/69336c58f87262bfd805aa9d45ccedf5bc54c149). There haven't been any relevant block changes since this tag so this should be safe.

Reply:
We discussed that ideally this would be templated and happen during "pxt bump" but not clear there's a mechanism to do this. One to revisit when we need to tag a version that doesn't exist!